.gitattributes

```diff
@@ -0,0 +1,4 @@
+*.pbf filter=lfs diff=lfs merge=lfs -text
```

Comment:
@mehdisadeghi what is this? I have never written such a git file 🤔  ...

Reply:
These are rules for Git Large Files. Github reads this file and stores large files out of the repo.